### DISCUSSION TITLE
Problem : "The model `gpt-4-0613` does not exist or you do not have access to it."

### DISCUSSION THREAD

**Discussion by lucasbenyattou** [lifan0127/ai-research-assistant#72] [Q&A]
Hello,

Sorry for the incovenience, but I've got a problem with the add-on. I downloaded Aria yesterday and I subscribed to GPT-4, but the message : "The model `gpt-4-0613` does not exist or you do not have access to it." still appears. I'm not very good at programming, so I don't know what the problem is.

Could you help me please ? 

Many thanks in advance,

**Comment by lifan0127** [comment]
Hello @lucasbenyattou, It seems that you do not have access to the GPT-4 model which needs to be set up in your OpenAI account. 

Please check out this OpenAI forum post for more detail: https://community.openai.com/t/openai-error-invalidrequesterror-the-model-gpt-4-does-not-exist-or-you-do-not-have-access-to-it/376230

Thank you!

**Comment by Liverace** [comment]
Is this because we must use a paid version of GPT-4 ?

**Reply by lifan0127** [reply]
Hi @Liverace I believe a paid OpenAI account (not ChatGPT subscription) is needed to access the GPT-4 model family as it is now.

**Reply by Liverace** [reply]
Thanks @lifan0127 Does it mean it becomes a prerequisite to use ARIA ?

**Reply by lifan0127** [reply]
I am afraid it is the case until OpenAI makes the GPT4 models more accessible.
